src/index.css

```diff
@@ -40,27 +32,39 @@ button {
   font-size: 1em;
   font-weight: 500;
   font-family: inherit;
-  background-color: #1a1a1a;
   cursor: pointer;
   transition: border-color 0.25s;
 }
-button:hover {
-  border-color: #646cff;
-}
+
 button:focus,
 button:focus-visible {
   outline: 4px auto -webkit-focus-ring-color;
 }
 
-@media (prefers-color-scheme: light) {
-  :root {
-    color: #213547;
-    background-color: #ffffff;
-  }
-  a:hover {
-    color: #747bff;
-  }
-  button {
-    background-color: #f9f9f9;
+svg,
+img {
+  max-width: 100%;
+  height: auto;
+  vertical-align: bottom;
+}
+
+#root {
+  padding: 20px;
```

Comment:
C'mooon. I told you not to do this) Please, apply some container.

Reply:
done